scripts/misc/generate_population_table.pl

```diff
@@ -410,55 +369,197 @@ sub get_size {
   return ($size == 0) ? '-' : $size;
 }
 
+# Alternate the background colour of the table rows
 sub set_bg {
   my $bg_class = shift;
   return ($bg_class eq '') ? ' class="bg2"' : '';
 }
 
+
+# Format the population description text
 sub parse_desc {
   my $content = shift;
-  #$content =~ s/<[^>]+>//g; # Remove URLs
   $content = '-' if (!$content);
   my @desc = split(/\.,/, $content);
   $content = "$desc[0]. $desc[1]." if scalar(@desc > 1);
   
-  if ($content =~ /(45 unrelated Han Chinese in Beijing, China, representing one of the populations studied in the International HapMap project)/) {
-    $content = "$1.";
-  }
-  
   return $content;
 }
 
+# Build a hash containing all the relevant information to create the species/projects/populations content
+# Mostly based on the vcf_config file and Variation databases
 sub get_project_populations {
-  my $projects = shift;
-  
+
   foreach my $project (@{$vcf_config->{'collections'}}) {
     my $project_id = $project->{'id'};
-    my $project_prefix = ($project->{'population_prefix'}) ? $project->{'population_prefix'} : '';
-    if ($projects->{$project_id}) {
+    next if ($project_id =~ /GRCh37/i);
```

Comment:
Why are you not using the assembly field for filtering by assembly?

Reply:
Good point!